### DISCUSSION TITLE
Pagination on Bootstrap 5 does not show the sentence of: Showing 1 to 25 of X results

### DISCUSSION THREAD

**Discussion by kotakomputer-ceo** [livewire/livewire#7604] [1. Help]
![image](https://github.com/livewire/livewire/assets/85127144/399b01f2-6631-499c-8439-f598660fd0ad)
Without Livewire, **Bootsrap 5** show the sentence of: **Showing 1 to 25 of X results** 
But on Livewire this Sentence not available.

**Comment by akamasAT** [comment]
yes it's missing, you have to publish the files and change them.
`php artisan livewire:publish --pagination`

add something like this where ever you want it:

`<div class="ps-2 align-self-center">

        <div class="text-muted fs-sm">
            <span>Show</span>
            <span class="fw-bolder">{{ $paginator->firstItem() }}</span>
            <span>to</span>
            <span class="fw-bolder">{{ $paginator->lastItem() }}</span>
            <span>of</span>
            <span class="fw-bolder">{{ $paginator->total() }}</span>
            <span>entries</span>
        </div>
</div>`
